.github/workflows/publish.yaml

```diff
@@ -55,10 +57,23 @@ jobs:
         shell: bash
         run: .github/scripts/output_build_info.sh ${{ matrix.distribution_path }}
 
+      - name: compress files (Windows)
+        if: runner.os == 'Windows'
+        run: |-
+          mkdir -p ${{ env.PUBLISH_DIR }}
+          cd ${{ matrix.distribution_path }}
+          powershell Compress-Archive -Path *.* -DestinationPath $Env:GITHUB_WORKSPACE/${{ env.PUBLISH_DIR }}/pedax-${{ runner.os }}.zip
```

Comment:
`Compress-Archive -Path .` -> contain root directory in zip
`Compress-Archive -Path *.*` -> not contain root directory in zip

Reply:
not correct.

**See: https://github.com/sensuikan1973/pedax/pull/83**